### DISCUSSION TITLE
Endroid or Bacon QRcode

### DISCUSSION THREAD

**Discussion by JohnDoeOffline** [RobThree/TwoFactorAuth#111] [General]
I find it peculiar that the creator of this project has chosen the most insecure way of generating the QR code via 'api.qrserver.com' (QRServerProvider) in the demo/example. Is there a working demo/example with the much more secure and local generated QRcode Endroid or Bacon?

**Comment by RobThree** [comment]
See https://github.com/RobThree/TwoFactorAuth/issues/104
